### ISSUE TITLE
Already at this stage

### ISSUE THREAD

**Issue by rafaelromcar-parabol** [ParabolInc/parabol#10410]
## Issue - Bug

- **How serious is the bug?** P2
- **OS:** macOS / Linux / Windows / ? -
- **Browser:** Chrome / Safari / Lynx / ? + version -
- **Node version:** `node --version` v20.11.0
- **NPM version:** `npm --version`

[ACTION-PRODUCTION-G8Z](https://parabol.sentry.io/issues/4573805927/?referrer=github_plugin)

We might have started seeing this error on Oct 9th at around 00:00, right after deploying v7.50.3 to production, but it's hard to tell as we keep only [14 days of Datadog logs](https://app.datadoghq.com/logs?query=-kube_namespace%3Akube-system%20env%3Aproduction%20-kube_namespace%3Adb-tools%20-kube_namespace%3Adatadog%20-kube_namespace%3Agmp-system%20service%3Aparabol%20%22Already%20at%20this%20stage%22&agg_m=count&agg_m_source=base&agg_t=count&cols=service%2Cpod_name%2Ckube_namespace&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1729341182521&to_ts=1731933182521&live=true)

```
SEND TO SENTRY Error: Already at this stage                                                                                                                                                                        
    at resolve (/home/node/parabol/dist/gqlExecutor.js:96530:53)                                                                                                                                                   
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)                                                                                                                                  
    at async Object.MutationnavigateMeetingResolver (/home/node/parabol/dist/gqlExecutor.js:126133:22)
```
### Acceptance Criteria (optional)

Users can:

- Do this
- Do that
- Cannot do that
  Triage is performed to find the root cause of the bug, timeboxed to ~1 hour.

**Estimated effort:** 1 hour to triage. More if root cause is already identified.

**Comment by Dschoordsch** [comment]
Sentry Issue: [ACTION-PRODUCTION-G8Z](https://parabol.sentry.io/issues/4573805927/?referrer=github_plugin)

**Comment by rafaelromcar-parabol** [comment]
Happened [5.55k times in the last two weeks](https://app.datadoghq.com/logs?query=-kube_namespace%3Akube-system%20env%3Aproduction%20-kube_namespace%3Adb-tools%20-kube_namespace%3Adatadog%20-kube_namespace%3Agmp-system%20%22Already%20at%20this%20stage%22&agg_m=count&agg_m_source=base&agg_t=count&cols=service%2Cpod_name%2Ckube_namespace&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1730390914020&to_ts=1731686914020&live=true). Fixing this would make our logs much easier to read

**Comment by rafaelromcar-parabol** [comment]
Hey there! I managed to reproduce this error on Staging. Very easy to do! Check my Loom https://www.loom.com/share/318404fe32434ce386fec7e4b144e8db

As I say on the video, that log line was introduced by @mattkrick during the NewMeeting Phase 3 migration. It might be useful for something, but I'm not quite sure for what, and right now it's adding a lot of noise. @mattkrick do you remember why you added that? Should we keep it?

**Comment by mattkrick** [comment]
oh that's great.
yeah, we should keep the line because it tells us that the client is sending messages to the server that aren't necessary, now that we know the root cause (thank you!) we can prevent the client from sending them in the first place

**Comment by mattkrick** [comment]
fixed in #10575
